doc/src/tutorial/matrices.rst

```diff
@@ -231,6 +231,163 @@ filled with `0`\ s.
     ⎢           ⎥
     ⎣0   0  0  5⎦
 
+
+Random Matrices
+---------------
+
+To create Matrices with specific properties use one of
+the random matrix generation functions, e.g.
+``invertible``, ``triangular``, ``orthogonal``.
+Find a :ref:`full list of random matrices <matrices-random>`.
+
+    >>> import random
+    >>> random.seed(1)
```

Comment:
A doctest should not modify the global random seed. Also there should be a better way to seed these functions.

Reply:
See e.g. https://github.com/sympy/sympy/issues/12628

Reply:
Thx for the hint.